src/deltacode/cli.py

```diff
@@ -91,3 +101,7 @@ def cli(new, old, json_file, all_delta_types):
 
     # generate JSON output
     write_json(deltacode, json_file, all_delta_types)
+
+    # show limited delta status
+    deltas_count = len([d for d in deltas(deltacode, all_delta_types)])
```

Comment:
I would like to see the counts of each particular Delta type, not the total number of deltas.

Reply:
So, you would like to see the counts of each type of "factor" for each delta? That is to say factors can be for example "added", "license info added", "moved" etc. and you would like to see the counts of these such factors, lets say we have 5 "added", 3 "moved" etc.

Reply:
Yes, I should have clarified that :) 

You can treat Deltas with multiple factors as their own entity for the sake of the counts

Also, I would be curious what the performance implications of generating these counts is. We may want to add this feature as a runtime option if it takes a long time on large datasets.